src/Fixer/AttributeNotation/GeneralAttributeRemoveFixer.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\AttributeNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\Fixer\ConfigurableFixerTrait;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolverInterface;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Analyzer\Analysis\AttributeAnalysis;
+use PhpCsFixer\Tokenizer\Analyzer\Analysis\NamespaceAnalysis;
+use PhpCsFixer\Tokenizer\Analyzer\Analysis\NamespaceUseAnalysis;
+use PhpCsFixer\Tokenizer\Analyzer\AttributeAnalyzer;
+use PhpCsFixer\Tokenizer\Analyzer\NamespacesAnalyzer;
+use PhpCsFixer\Tokenizer\Analyzer\NamespaceUsesAnalyzer;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Raffaele Carelle <raffaele.carelle@gmail.com>
+ *
+ * @implements ConfigurableFixerInterface<_AutogeneratedInputConfiguration, _AutogeneratedComputedConfiguration>
+ *
+ * @phpstan-import-type _AttributeItems from AttributeAnalysis
+ * @phpstan-import-type _AttributeItem from AttributeAnalysis
+ *
+ * @phpstan-type _AutogeneratedInputConfiguration array{
+ *  attributes?: list<string>
+ * }
+ * @phpstan-type _AutogeneratedComputedConfiguration array{
+ *  attributes: list<string>
+ * }
+ */
+final class GeneralAttributeRemoveFixer extends AbstractFixer implements ConfigurableFixerInterface
+{
+    /** @use ConfigurableFixerTrait<_AutogeneratedInputConfiguration, _AutogeneratedComputedConfiguration> */
+    use ConfigurableFixerTrait;
+
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Configured attributes should be omitted (FQNs strings).',
```

Comment:
Personally I wouldn't follow `general_phpdoc_annotation_remove `'s "omitted" convention, as "omit" is like "skip", while the real intention here is to _remove_ the attribute(s). Also "should" does not fit here, as this is a _must_ when configured.

```suggestion
            'Removes configured attributes by their respective FQN.',
```

Or something like that.

Reply:
Hello @Wirone, thanks for your feedback. Fixed.

Reply:
pls update for other `general*` fixer(s) in that case :D

Reply:
sorry @keradus, I didn't get it what I should do

Reply:
could you apply Wirone's request to those files too?
```
src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php:            'Configured annotations should be omitted from PHPDoc.',
src/Fixer/Phpdoc/PhpdocNoAccessFixer.php:            '`@access` annotations should be omitted from PHPDoc.',
src/Fixer/Phpdoc/PhpdocNoEmptyReturnFixer.php:            '`@return void` and `@return null` annotations should be omitted from PHPDoc.',
src/Fixer/Phpdoc/PhpdocNoPackageFixer.php:            '`@package` and `@subpackage` annotations should be omitted from PHPDoc.',
```

Reply:
ok, done.